sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageBuilder.java

```diff
@@ -33,15 +34,15 @@ public AzureBlobStorageBuilder url(String url) {
     /*
      * Specifies the version of the operation to use for this request.
      */
-    private String version;
+    private BlobServiceVersion version;
```

Comment:
Don't modify this, it is codegen, same goes for all the Impl class changes. Instead toString the enum before using it.